lms/djangoapps/organizations/management/commands/tests/test_move_organizations_entries.py

```diff
@@ -40,7 +41,7 @@ def setUp(self):
             org.groups.add(group1, group2)
             org.workgroups.add(workgroup)
 
-
+    @skip
```

Comment:
If this test is no longer valid/applicable, then I would say we should just remove it, along with the script that it was validating -- thoughts?